src/visualization/plot_recipes.jl

```diff
@@ -2,7 +2,7 @@
 # Since these FMAs can increase the performance of many numerical algorithms,
 # we need to opt-in explicitly.
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
-@muladd begin
+# @muladd begin
```

Comment:
Why did you comment this out?

Reply:
Oops - that wasn't an intentional commit.

The macros resulted in more struct redefinition errors, so I commented them out during development. I'll add them back in soon.